### ISSUE TITLE
Questions re: updates section

### ISSUE THREAD

**Issue by niryariv** [Gizra/IBLI#83]
1. Where do the items in News come from? Entered by admin on the site, or some external feed?
2. Same question regarding Events - is this a Google Calendar or similar feed?
3. What should be the tab order of News / Events / Partners?

cc @DavidGizra @NofarGizra

**Comment by bnedavies** [comment]
closing...addressed in #153
